README.md

```diff
@@ -8,7 +8,7 @@
 be used separately or in combination for the Discord API.
 
 The ecosystem of first-class crates includes `twilight-cache`,
-`twilight-command-parser`, `twilight-gateway`, `twilight-http`, `twilight-model`,
+`twilight-command-parser`, `twilight-gateway`, `twilight-http`, `twilight-model`, `twilight-standby`
```

Comment:
This looks like there was some kind of mistake rebasing maybe? Regardless I don't think this change belongs in this pr.

Reply:
i realized it was missing while i was documenting the the simd-json flag. i will remove it now